core/shared/src/main/scala/scalaz/zio/internal/FiberContext.scala

```diff
@@ -374,6 +404,30 @@ private[zio] final class FiberContext[E, A](
 
   final def poll: UIO[Option[Exit[E, A]]] = ZIO.effectTotal(poll0)
 
+  final def inheritLocals: UIO[Unit] =
+    for {
+      descriptor <- ZIO.descriptor
+      locals     <- copyFiberLocals(descriptor)
+      _ <- ZIO.foreach_(locals) {
+            case (fiberRef, (value, updateFiberId)) =>
+              if (updateFiberId != descriptor.id) {
+                new zio.ZIO.FiberRefSet[Any](
+                  fiberRef.asInstanceOf[FiberRef[Any]],
+                  value,
+                  descriptor.id
+                )
+              } else {
+                ZIO.unit
+              }
+          }
+    } yield ()
+
+  private def copyFiberLocals(descriptor: Descriptor) =
+    UIO.effectTotal {
```

Comment:
`descriptor` is not used in this function.

Reply:
removed